lms/djangoapps/certificates/management/commands/rescind_cert.py

```diff
@@ -0,0 +1,104 @@
+    help = u'Rescind a certificate for a particular user in a particular course.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-c', '--course',
+            metavar='COURSE_ID',
+            dest='course',
+            default=False,
+            help='The course id (e.g., mit/6-002x/circuits-and-electronics) of the course in which the '
+                 'certificate of the student should be updated',
+        ),
+        make_option(
+            '-u', '--user',
+            metavar='USERNAME',
+            dest='username',
+            default=False,
+            help='The username or email address for whom grading and certificate status should be updated',
+        ),
+        make_option(
+            '-s', '--status',
+            metavar='STATUS',
+            dest='status',
+            default=CertificateStatuses.unavailable,
+            help='The status to set to the corresponding certificate',
+        ),
+    )
+
+    def handle(self, *args, **options):
+
+        if options['course']:
+            # try to parse out the course from the serialized form
+            try:
+                course_id = CourseKey.from_string(options['course'])
+            except InvalidKeyError:
+                course_id = SlashSeparatedCourseKey.from_deprecated_string(options['course'])
+        else:
+            raise CommandError(u'You must specify a course')
+
+        user = options['username']
+        if not user:
+            raise CommandError(u'The student\'s username is required')
+
+        status = options['status']
+        valid_statuses = [
+            CertificateStatuses.unavailable,
+            CertificateStatuses.downloadable,
+            CertificateStatuses.notpassing,
+        ]
+        if status not in valid_statuses:
+            raise CommandError(
+                u'INVALID STATUS. '
+                'Supported Statuses: `unavailable`, `downloadable`, `notpassing`'
+            )
+
+        if '@' in user:
+            student = User.objects.get(email=user, courseenrollment__course_id=course_id)
+        else:
+            student = User.objects.get(username=user, courseenrollment__course_id=course_id)
+        try:
+            certificate = GeneratedCertificate.objects.get(user=student, course_id=course_id)
+            certificate.status = status
+            certificate.save()
+            cert_saved_msg = (
+                u"""The certificate status for student {student_id}
+                in course '{course_id}'
+                has been set to '{status}'""".format(
+                    student_id=student.id,
+                    course_id=course_id,
+                    status=certificate.status,
+                )
+            )
+            LOGGER.info(cert_saved_msg)
+            print cert_saved_msg
+        except GeneratedCertificate.DoesNotExist:
+            LOGGER.error(
+                (
+                    u"""Certificate for student %d
```

Comment:
docstring style comments should only be used for docstrings.

In this case, the whitespace would presumably be preserved, yielding:

```
Certificate for student XXX
                         in course YYY does not exist
```

What we would prefer to log is:

```
Certificate for student XXX in course YYY does not exist
```

which we can achieve by using "normal" strings.

If the string is long enough that it warrants splitting (may not be the case here),
we should use implied string concatenation [1]:

```python
LOGGER.error(
    (
        "Certificate for student %d "
        "in course %s does not exist"
    ),
    student.id,
    course_id,
)
```


[1] https://docs.python.org/2/reference/lexical_analysis.html#string-literal-concatenation